mallard/client.py

```diff
@@ -192,7 +192,7 @@ async def search(self, query, message):
 
         # pylint: disable=assigning-non-slot
         embed = discord.Embed(type="rich")
-        embed.timestamp = datetime.now()
+        embed.timestamp = datetime.utcnow()
```

Comment:
does this work? (we'll need to import `datetime.timezone`)
```suggestion
        embed.timestamp = datetime.now(timezone.utc)
```
https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

Reply:
both would work but there is no benefit of doing this over utcnow